community/mpdscribble/APKBUILD

```
@@ -1,55 +1,37 @@
-# Contributor: 
+# Contributor: Leo <thinkabit.ukim@gmail.com>
 # Maintainer: 
 pkgname=mpdscribble
 pkgver=0.22
-pkgrel=1
+pkgrel=2
 pkgdesc="last.fm client for mpd"
-url="http://mpd.wikia.com/wiki/Client:Mpdscribble"
+url="https://github.com/MusicPlayerDaemon/mpdscribble"
 arch="all"
-license="GPL"
-depends=""
-depends_dev=""
-makedepends="$depends_dev libmpdclient-dev curl-dev glib-dev"
-install=""
-subpackages="$pkgname-doc"
+license="GPL-2.0-or-later"
+makedepends="libmpdclient-dev curl-dev glib-dev automake libtool autoconf"
+subpackages="$pkgname-doc $pkgname-openrc"
 source="https://www.musicpd.org/download/mpdscribble/${pkgver}/mpdscribble-${pkgver}.tar.bz2
 	mpdscribble.initd"
 
-_builddir="$srcdir"/mpdscribble-$pkgver
 prepare() {
-	local i
-	cd "$_builddir"
-	update_config_sub || return 1
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
+	autoreconf -fi
```

```suggestion
	default_prepare
	autoreconf -fi
```
Even though there are no patches to be applied, we always call `default_prepare` when overriding the default function.

---

done